packages/animations/example/android/.pluginToolsConfig.yaml

```diff
@@ -0,0 +1,3 @@
+buildFlags:
+  _pluginToolsConfigGlobalKey:
+    - "--no-tree-shake-icons"
```

Comment:
Do all of these need EOF newlines? Or is github complaining unnecessarily? :)

Reply:
They all needed new lines. Updated.